File: Algorithms 2021/February2021/week1/week1.py
# Friday February 12------------------------------------------------
# Print the sum of the array's elements as a single integer.
def simpleArraySum(ar):
    sum = 0
    for i in range(len(ar)):
        sum = sum + ar[i]

    return sum


thislist = [1, 2, 3, 4, 5, 6, 7]
simpleArraySum(thislist)


# Complete the aVeryBigSum function below.
def aVeryBigSum(ar):
    sum = 0
    for i in range(len(ar)):
        sum = sum + ar[i]
    return sum

File: Algorithms 2021/February2021/week1/week1.js
// Friday February 12------------------------------------------------
// Print the sum of an array
const simpleArraySum = (ar) =>{
    var sum = 0;
    for(var i = 0; i < ar.length; i++){
        sum = sum + ar[i];
    }
     return sum;
}

let thisList = [1,2,4,5,6,7,8]

simpleArraySum(thisList)



// Complete the aVeryBigSum function below.
const aVeryBigSum = (ar) => {
    var sum = 0;
    for(var i = 0; i < ar.length; i++){
        sum = sum + ar[i];  
    }
    
  return sum;

}

var bigNum = [1000000001, 1000000002, 1000000003, 1000000004, 1000000005]
aVeryBigSum(bigNum);

